build.py

```diff
@@ -210,6 +210,10 @@ def gen_core(self):
         ("warning_level", "DEFAULT"),
       ]
 
+    # Add Closure provide, require and inherits methods.
+    params.append(("js_code", """var goog=goog||{};goog.provide=function(){};goog.require=function(){};
```

Comment:
I don't see any goog.provide or goog.require in the compressed files.  Why are these needed?

Reply:
It's there: https://github.com/google/blockly/blob/master/blockly_compressed.js#L7
We need it for our message files as they have provide/require calls eg: [here](https://github.com/google/blockly/blob/master/msg/js/en.js). and are included as is in html files eg: [here](https://github.com/google/blockly/blob/master/demos/fixed/index.html#L8)

Reply:
We also have calls to require/provide in our custom fields in demos, but I know we can get rid of those. I'm just not sure about the message files, does Blockly games need the call to require/provide in order to bundle message files?

Reply:
Discussed offline, we'll go ahead and remove the require/provide calls in the built message files.